middleware/fileserver_test.go

```diff
@@ -197,6 +210,12 @@ func beforeServeHTTPTest(t *testing.T) {
 			return
 		}
 		f.Close()
+
+		// and set the last modified time
+		err = os.Chtimes(absFile, fixedTime, fixedTime)
```

Comment:
A better option (likely) would be to structure a test case on something similar to "failingFS", where the returned modification time is constant.